apis/infix-en_GB/atrium-api-infix-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/infix/en_GB/ListFeatureAssertionsSpec.kt

```diff
@@ -0,0 +1,44 @@
+package ch.tutteli.atrium.api.infix.en_GB
+
+import ch.tutteli.atrium.creating.Expect
+import ch.tutteli.atrium.specs.feature1
+import ch.tutteli.atrium.specs.fun2
+import ch.tutteli.atrium.specs.notImplemented
+
+class ListFeatureAssertionsSpec : ch.tutteli.atrium.specs.integration.ListFeatureAssertionsSpec(
+    feature1<List<Int>, Int, Int>(Expect<List<Int>>::get),
+    fun2<List<Int>, Int, Expect<Int>.() -> Unit>(Expect<List<Int>>::get),
+    feature1<List<Int?>, Int, Int?>(Expect<List<Int?>>::get),
+    fun2<List<Int?>, Int, Expect<Int?>.() -> Unit>(Expect<List<Int?>>::get)
+) {
+
+    companion object {
+
+        @Suppress("unused", "UNUSED_VALUE")
+        private fun ambiguityTest() {
+
+            val a1: Expect<List<Int>> = notImplemented()
+            val a2: Expect<out List<Int>> = notImplemented()
+            val a1b: Expect<List<Int?>> = notImplemented()
+            val a2b: Expect<out List<Int?>> = notImplemented()
+
+            val a3: Expect<out List<*>> = notImplemented()
+
+            a1 get 1
```

Comment:
please change the structure, first a1 and a2 then a1b and a2b and finally a3 (see ch.tutteli.atrium.api.fluent.en_GB.ListFeatureAssertionsSpec) this should make it easier to add further functionality in an ordered manner